docs/public/SUMMARY.md

```diff
@@ -0,0 +1,110 @@
+- [Installation]()
+    - [Prerequisites](installation/prerequisites.md)
+    - [Zonemaster-LDNS](installation/zonemaster-ldns.md)
+    - [Zonemaster-Engine](installation/zonemaster-engine.md)
+    - [Zonemaster-CLI](installation/zonemaster-cli.md)
+    - [Zonemaster-Backend](installation/zonemaster-backend.md)
+    - [Zonemaster-GUI](installation/zonemaster-gui.md)
+    - [Docker](installation/docker.md)
+- [Upgrade]()
+    - [Backend](upgrade/backend.md)
+- [Configuration]()
+    - [Profiles](configuration/profiles.md)
+    - [Backend](configuration/backend.md)
+        - [Environment Variables](configuration/backend-environment-variables.md)
+    - [GUI](configuration/gui.md)
+- [Using]()
+    - [CLI](using/cli.md)
+    - [RPCAPI](using/getting-started.md)
+        - [RPCAPI Reference](using/backend-api.md)
```

Comment:
Following the comment above, this could be `backend-rpcapi-reference.md`.

Reply:
New "backend" folder -> "using/backend/rpcapi-reference.md"